src/explanations/dai.yaml

```diff
@@ -0,0 +1,82 @@
+
+
+    DAI is used in various ways, including as a medium of exchange, a store of value, and a collateral asset. It has gained widespread adoption in the cryptocurrency community and has been used in a number of decentralized finance (DeFi) applications.
+
+
+    ### Minting and Burning DAI
+
+
+    Dai is generated when users deposit collateral assets into [Maker Vaults](https://makerdao.com/en/whitepaper/#maker-vaults) (Maker Vaults facilitate the generation of Dai against locked-up collateral. Users open Vaults via the Oasis Borrow dashboard or one of many other user interfaces that allow the management of Vault operations ) within the Maker Protocol. This process allows users to access liquidity and enters Dai into circulation. Others can obtain Dai by purchasing it from brokers or exchanges, or by receiving it as payment.
+
+
+    Once generated, purchased, or received, Dai can be used in the same way as other cryptocurrencies. It can be sent to others, used as payment for goods and services, and even saved through a feature of the Maker Protocol called the [Dai Savings Rate (DSR)](https://ethereumprice.org/guides/article/dai-savings-rate-explained/).
+
+
+    If a user who has borrowed DAI against collateral decides to pay back the loan, the DAI coins are "burned" and the collateral deposit in the account is available for withdrawal. The process of minting and burning DAI helps to maintain its value at or near one US dollar.
+
+
+    The MCD_DAI_JOIN adapter contract in the Maker Protocol is authorized to call the mint and burn functions of the DAI token in order to increase or decrease the total supply of DAI. These functions are also used when users deposit or withdraw DAI from the Dai Savings Rate contract.
+
+
+    ### Transparency and Audits
+
+
+    MakerDAO and the DAI stablecoin are designed to be transparent and auditable. The smart contracts that govern the Collateralized Debt Positions (CDPs) and the minting and burning of DAI are open-source, which allows anyone to review the code and understand how the system works.
+
+
+    In addition, MakerDAO has undergone several independent audits by reputable firms, which have found that the system is secure and functioning as intended. In 2019, Callisto Network Security conducted a security audit of the Dai Token (DAI) smart contract which identified four issues, including three of low severity and one of owner privileges (which could potentially be risky for investors). No critical security issues were discovered. The audit specifically identified known vulnerabilities of ERC-20 tokens, the ability for the contract owner to block transferring, a missing event in ERC20 compliance, and the need to check input addresses for empty values. Overall, the audited smart contract was determined to be safe to deploy, with only low severity issues found during the audit. [Here](https://callisto.network/dai-token-dai-security-audit/) is the complete report.
```

Comment:
No Audits done post 2019?

Reply:
I could not find any audit reports other than the one by Callisto Network